reconcile/cli.py

```diff
@@ -1906,8 +1907,21 @@ def integrations_manager(
         reconcile.integrations_manager,
         ctx.obj,
         environment_name,
+        get_integration_cli_meta(),
         thread_pool_size,
         internal,
         use_jump_host,
         image_tag_from_ref,
     )
+
+
+def get_integration_cli_meta() -> dict[str, IntegrationMeta]:
+    integration_meta = {}
+    for integration_name in integration.list_commands(None):  # type: ignore
```

Comment:
That's some tricky meta programming. Wonder if it should actually be in the schema? Please at least add some comments.

Reply:
i left a comment